.github/workflows/release.yaml

```diff
@@ -0,0 +1,18 @@
+on:
+  push:
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: Create Release
+
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 10
```

Comment:
I would suggest increasing the fetch depth.  I think 50 would probably be sufficient, 10 is too smal to guarantee all changes since the previous release have been parsed.